spec/requests/search_request_spec.rb

```diff
@@ -131,10 +127,36 @@
   context "blank search, with different columns" do
     let(:search) { nil }
     let(:fields) { "fields%5B%5D=citation" }
-    let(:agency) { nil }
 
     it "succeeds" do
       expect(response.successful?).to be_truthy
     end
   end
+
+  context "publication date search" do
+    before { create :sorn, system_name: "NEW SORN", publication_date: "2019-01-13", citation: "different citation" }
+
+    it "only returns the newer sorn in date range" do
+      get "/search?starting_year=2019"
+
+      expect(response.body).to include "NEW SORN" # Newer sorn date
+      expect(response.body).to include "2019-01-13" # Newer sorn date
+      expect(response.body).not_to include "2000-01-13" # Older sorn date
+    end
+
+    it "only returns the older sorn in date range" do
+      get "/search?ending_year=2001"
+
+      expect(response.body).to include "2000-01-13" # Older sorn date
+      expect(response.body).not_to include "NEW SORN"
+      expect(response.body).not_to include "2019-01-13" # Newer sorn date
+    end
+
+    it "ending year is inclusive" do
```

Comment:
i could go either way, but would it make sense to have a test case to just check that the 2 search params can be on the `/search` endpoint?